### PR TITLE
Makes plugin compatible with pulpcore-plugin 0.1.0rc4

### PR DESCRIPTION
**Pull request by dkliban**
This PR contains multiple commits that were needed to get this plugin compatible with pulpcore-plugin 0.1.0rc4. 

This PR also introduces the template_config.yml that is needed to manage the Travis config with plugin-template.

### CODE REVIEW

File: .travis/check_commit.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  exit 0
+fi
+
+if [ "$TRAVIS_COMMIT_RANGE" != "" ]; then
+  RANGE=$TRAVIS_COMMIT_RANGE
+elif [ "$TRAVIS_COMMIT" != "" ]; then
+  RANGE=$TRAVIS_COMMIT
+fi
+
+# Travis sends the ranges with 3 dots. Git only wants one.
```
Comment (mdellweg) [inline .travis/check_commit.sh]:
```suggestion
# Travis sends the ranges with 3 dots. Git only wants two.
```

Comment (mdellweg) [reply .travis/check_commit.sh]: Alternatively: "Git only knows why."

Comment (dkliban) [reply .travis/check_commit.sh]: lol

File: .travis/install.sh [hunk 2]
```diff
@@ -1,36 +1,23 @@
...
-pip install -r test_requirements.txt
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by bootstrap.py. Please use
+# bootstrap.py to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
 
-pushd ..
+set -v
```
Comment (mdellweg) [inline .travis/install.sh]: I think, this should at least be `set -e` here and elsewhere.
I prefer `set -euv`.

File: .travis/before_script.sh [hunk 3]
```diff
@@ -1,14 +1,26 @@
...
+#!/usr/bin/env sh
+
+# WARNING: DO NOT EDIT!
+#
+# This file was generated by plugin_template, and is managed by bootstrap.py. Please use
+# bootstrap.py to update this file.
+#
+# For more info visit https://github.com/pulp/plugin_template
+
+set -v
```
Comment (mdellweg) [inline .travis/before_script.sh]: There's another one

**Pull request merged**